functions/tx-manager_dashboard/main.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import unicode_literals
```

Comment:
Why is the tx-manager_dashboard function being added in this PR? Shouldn't this be a separate Issue and PR?